manifest/docker_schema2.go

```diff
@@ -215,6 +216,19 @@ func (m *Schema2) LayerInfos() []LayerInfo {
 	return blobs
 }
 
+// imgInspectLayersFromLayerInfos converts a list of layer infos, presumably obtained from a Manifest.LayerInfos()
+// method call, into a format suitable for inclusion in a types.ImageInspectInfo structure.
+func imgInspectLayersFromLayerInfos(infos []LayerInfo) []types.LayerDetail {
```

Comment:
When this function is used in all the format handlers, `manifest/common.go` would be a better location.